### DISCUSSION TITLE
Facing Error While installing Dependencies !!

### DISCUSSION THREAD

**Discussion by ayushgupta4002** [RocketChat/Rocket.Chat#30320] [Q&A]
Hi Folks; My Node Version  and  meteor version align with the one mentioned in docs! Still am not able to install dependencies! Kindly help me out ! 
![image](https://github.com/RocketChat/Rocket.Chat/assets/66496371/e020d550-be68-4e7e-9793-fc8d8372f08f)

**Comment by aryan-bhokare** [comment]
Hey is the issue solved??

**Reply by v-a14** [reply]
I am facing the similar issue, can you please help ?

**Comment by aryan-bhokare** [comment]
use proper node version
